_plugins/ganalytics.md

```diff
@@ -12,3 +12,9 @@ info:
 Google Analytics for Pico with basic options.
 
 This plugin will automatically add the Google Analytics tracking script to the < head > section of each page generated by Pico.
+
+**Features**
+
+* Pico v.1.0 - ready!
```

Comment:
This actually should be obligatory to get listed on this page :laughing: (Nevertheless it isn't quite true at the moment, however, we shouldn't mention this explicitly)